content/javascript/concepts/objects/terms/fromEntries/fromEntries.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.fromEntries()'
+Description: 'Returns a new object with properties from a given iterable.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.fromEntries()`** method returns a new object with properties from a given list of key-value pairs.
+
+## Syntax
+
+```pseudo
+Object.fromEntries(iterableObject);
+```
+
+The `iterableObject` is usually an Array or a Map. A single element in the iterable object can be an array with two elements. The first element in this array will be the `key` and the second element will be the `value` for a single property in the returned object.
```

Comment:
Here is where I added the explanation of the first element being a `key` and the second element being a `value`.

Reply:
@Jamesllllllllll In my opinion, it makes more sense to move this above, as it doesn't have much to do with the syntax but more with how the `.fromEntries()` method works.

Reply:
@ishg-153 Sounds good!